pkg/destroy/gcp/gcp.go

```diff
@@ -261,7 +302,7 @@ func newPendingItemTracker() pendingItemTracker {
 }
 
 // GetAllPendintItems returns a slice of all of the pending items across all types.
-func (t pendingItemTracker) GetAllPendingItems() []cloudResource {
+func (t *pendingItemTracker) GetAllPendingItems() []cloudResource {
```

Comment:
I don't see a valid reason for not having this be a pointer receiver - right now you get away with it since the map is implicitly a pointer type, but it makes everything a massive pain (I'd need to define a pointer type for my slice ... )